testing/scenario_app/android/app/src/main/java/dev/flutter/scenarios/PlatformViewsActivity.java

```diff
@@ -8,8 +8,12 @@
 import io.flutter.embedding.engine.FlutterEngine;
 
 public class PlatformViewsActivity extends TestActivity {
+  // WARNING: These strings must all be exactly the same length to avoid
+  // breaking the 'create' method's manual encoding in the test. See the
+  // TODO(stuartmorgan) about encoding alignment in platform_view.dart
```

Comment:
I lost at least an hour to figuring out why my new factory wasn't working at all 😐  I didn't want to yak-shave right now since this is an important fix, and one that I want to scope-limit so that we can cherry-pick it to stable, so I just left comments so that at least the problem will be obvious to the next person.